.github/workflows/test.yml

```diff
@@ -82,3 +82,151 @@ jobs:
 
       - name: Run Tests
         run: ./vendor/bin/sail artisan test
+
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/main')
+    needs: [ test ]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Extract short commit hash
+        run: |
+          echo "::set-env name=COMMIT::$(echo ${GITHUB_SHA} | cut -c1-7)"
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+
+      - name: Extract tag
+        run: |
+          echo "::set-env name=TAG::$(git describe --tags --abbrev=0)"
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+
+      - name: Set deployment variables
+        uses: allenevans/set-env@v2.0.0
+        with:
+          APP_URL: https://ecfmp.vatsim.net
+          APPLICATION_ROOT: ${{ secrets.APPLICATION_ROOT }}
+          RELEASE_DIRECTORY: ${{ secrets.APPLICATION_ROOT }}/releases/${{ env.COMMIT }}
+          SHARED_DIRECTORY: ${{ secrets.APPLICATION_ROOT }}/shared
+          PHP_PATH: /usr/bin/php
+
+      - name: GitHub Deployment
+        uses: chrnorm/deployment-action@releases/v1
+        id: github_deployment
+        with:
+          token: ${{ github.token }}
+          target_url: https://ecfmp.vatsim.net
+          environment: production
+          ref: ${{ env.COMMIT }}
+
+      # Everything to do with Yarn
+      - name: Setup Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16'
+
+      - name: Get Yarn Cache Directory
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Restore Yarn Cache
+        uses: actions/cache@v1
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Yarn Install
+        run: yarn
+
+      - name: Yarn Run
+        run: yarn run prod

```

Comment:
We're currently using NPM. This isn't a big issue right now (I don't often change these depencencies). However, I recommend you also run `yarn`, remove `package-lock.json`, and add `yarn.lock`.

Alternative is to change this bit to use NPM instead.

Reply:
Yep - copy pasta from another workflow I have so forgot to change Yarn to NPM, should be changed now :)